### PR TITLE
Issue 2828510: Prevent installation of Commerce if bcmath PHP extensi…

### PR DESCRIPTION
**Pull request by mitrpaka**
…on missing

### CODE REVIEW

**Comment by mglaman** [comment]
@mitrpaka this is true, however were you able to run `composer install` with `bcmath` missing?

**Comment by bojanz** [comment]
We're missing an ext-bcmath: "*" requirement in the composer.json, in addition to the code in this PR

File: commerce.install [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+/**
+* Implements hook_requirements().
+*/
+function commerce_requirements($phase) {
+  $requirements = [];
+  if ($phase == 'install') {
+    if (!extension_loaded('bcmath')) {
+      $requirements['commerce'] = [
+      'description' => t("Commerce requires you to enable the following PHP extension: bcmath"),
```
Comment (bojanz) [inline commerce.install]: Wrong indentation for 'description' and 'severity'. The error message can be inside single quotes. The message should be 'Commerce requires the bcmath PHP extension.'

**Review by bojanz — CHANGES_REQUESTED** [review]

**Comment by mglaman** [comment]
@bojanz et @mitrpaka I went ahead and addressed those issues on the PR in @46d00b002c07f8dbd8ee7b64277299a8138c6965

**Pull request merged**